myeongseok/db/migrations/20221104143357_update_posts_table.sql

```diff
@@ -0,0 +1,5 @@
+-- migrate:up
+ALTER TABLE posts ADD post_image varchar(1000) NOT NULL AFTER title
```

Comment:
```suggestion
ALTER TABLE posts ADD post_image_url varchar(1000) NOT NULL AFTER title
```
image 파일을 저장하는 것이 아니라 주소를 저장하는 것이기 때문에 정확히 적어주는 것이 중요합니다~

Reply:
네 감사합니다! dbmate 수정 및 해당 column이 사용된 query문의 column 이름도 수정하였습니다!